smallboard/templates/base.html

```diff
@@ -27,7 +27,12 @@
                 <a class="nav-link" href="{% url 'logout' %}">Logout</a>
             </li>
             <li class="nav-item">
-                <div class="nav-link disabled">Logged in as {{ user.username }}
+                <div class="nav-link disabled">Logged in as
+            {% if user.first_name %}
```

Comment:
Nit: Should we check for both first_name and last_name?

Reply:
The last_name just won't show if it's empty. No harm done I think.